### PR TITLE
Fix data entry errors pointed out in issues #37 and #38

### PR DESCRIPTION
**Pull request by frreiss**
This PR includes fixes for the following errors:
* `MISC` tag misspelled as `MIC` (issue #37; commit 8c0320c)
* `CHINA` corrected from `PER` to `LOC` instead of from `PER` to `ORG` (issue #38; commit 2035e84)
* Additional sports teams not corrected from `LOC` to `ORG` in test fold document 0 (issue #38; commit 2035e84)

### CODE REVIEW

File: corrected_labels/all_conll_corrections_combined.csv [hunk 1]
```diff
@@ -351,11 +351,19 @@ test,0,"[1072, 1077): 'Syria'",LOC,Tag,"[1072, 1077): 'Syria'",ORG,Soccer team
...
 test,0,"[196, 201): 'Syria'",LOC,Tag,"[196, 201): 'Syria'",ORG,Soccer team
+test,0,"[2056, 2059): 'UAE'",LOC,Tag,"[2056, 2059): 'UAE'",ORG,Soccer team; see issue #38
+test,0,"[2065, 2071): 'Kuwait'",LOC,Tag,"[2065, 2071): 'Kuwait'",ORG,Soccer team; see issue #38
+test,0,"[2076, 2087): 'South Korea'",LOC,Tag,"[2076, 2087): 'South Korea'",ORG,Soccer team; see issue #38
+test,0,"[2096, 2105): 'Indonesia'",LOC,Tag,"[2096, 2105): 'Indonesia'",ORG,Soccer team; see issue #38
 test,0,"[249, 254): 'China'",LOC,Tag,"[249, 254): 'China'",ORG,Soccer team
+test,0,"[363, 373): 'Uzbekistan'",LOC,Tag,"[363, 373): 'Uzbekistan'",ORG,Soccer team; see issue #38
 test,0,"[375, 380): 'China'",LOC,Tag,"[375, 380): 'China'",ORG,Soccer team
-test,0,"[40, 45): 'CHINA'",PER,Tag,"[40, 45): 'CHINA'",LOC,
+test,0,"[40, 45): 'CHINA'",PER,Tag,"[40, 45): 'CHINA'",ORG,Fred: Soccer team
```
Comment (xuhdev) [inline corrected_labels/all_conll_corrections_combined.csv:366]: Is "Fred: " supposed to be here?

Comment (BryanCutler) [reply corrected_labels/all_conll_corrections_combined.csv:366]: I think that is just his comment about the tag right?

**Review by BryanCutler — APPROVED** [review]
LGTM

File: corrected_labels/all_conll_corrections_combined.csv (hunk 1 above)
Comment (kmh4321) [reply corrected_labels/all_conll_corrections_combined.csv:366]: I think it's okay, that's the comments field anyway

**Review by kmh4321 — APPROVED** [review]

File: corrected_labels/all_conll_corrections_combined.csv (hunk 1 above)
Comment (frreiss) [reply corrected_labels/all_conll_corrections_combined.csv:366]: I've been marking all the comments from these later review passes with "Fred:" to distinguish them from the original labelers' comments.

**Pull request merged**